DS13/code/modules/EMH/EMH.dm

```diff
@@ -0,0 +1,291 @@
+	var/poll_message = "Do you want to play as an emergency medical hologram?"
+	var/list/mob/dead/observer/candidates = pollCandidatesForMob(poll_message, ROLE_EMH, null, FALSE, 100, M)
+	for(var/mob/dead/observer/S in candidates)
+		if(S.key in GLOB.EMH_blacklist)
+			candidates -= S
+	if(LAZYLEN(candidates))
+		var/mob/dead/observer/C = pick(candidates)
+		to_chat(M, "Your mob has been taken over by a ghost!")
+		message_admins("[key_name_admin(C)] has taken control of ([ADMIN_LOOKUPFLW(M)])")
+		M.ghostize(0)
+		M.key = C.key
+		return TRUE
+
+
+/obj/machinery/emh_emitter/proc/is_occupied()
+	var/mob/living/global_emh = null //For deactivation of AFK EMHs
+	global_emh = emh
+	if(!emh)
+		var/mob/living/carbon/human/species/holographic/S = locate(/mob/living/carbon/human/species/holographic) in GLOB.alive_mob_list
+		if(istype(S) && !QDELETED(S))
+			global_emh = S
+	if(global_emh && !QDELETED(global_emh))
+		if(global_emh.mind && global_emh.client)
+			return TRUE
+		else
+			return FALSE
+	if(EMH_present())
+		return TRUE
+	return FALSE
+
+//SPECIES AND OUTFIT
+
+/datum/species/holographic
+	name = "Hologram"
+	id = "human" //Because we need an emergency reference (for such like limbs).
+	default_color = "FFFFFF"
+	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS)
+	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
+	use_skintones = 1
+	skinned_type = null
+	species_traits = list(NOBLOOD)
+	inherent_traits = list(TRAIT_RADIMMUNE,TRAIT_RESISTHEAT,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_RADIMMUNE,TRAIT_NOFIRE,TRAIT_PIERCEIMMUNE,TRAIT_NOHUNGER,TRAIT_NODISMEMBER, TRAIT_PACIFISM,TRAIT_POOR_AIM)//EMHs are not programmed to fire weapons / cause harm
+	inherent_biotypes = list(MOB_HUMANOID)
+	meat = null
+	mutantlungs = null
+	mutantliver = null
+	mutantstomach = null
+	brutemod = 2
+	heatmod = 2
+	breathid = "" //You don't need air, you're a hologram.
+	damage_overlay_type = ""//You're a hologram, you don't bleed.
+
+/datum/species/holographic/spec_death(gibbed = FALSE, mob/living/carbon/human/H)
+	H.unequip_everything() //Don't spawn in as an EMH, rush the spare, then suicide so it deletes
+	playsound(H.loc, 'DS13/sound/effects/holofizzle.ogg', 100, 0)
+	H.visible_message("[H] fizzles away quietly...")
+	qdel(H)
+	. = ..()
+
+/mob/living/carbon/human/species/holographic
+	race = /datum/species/holographic
+
+/datum/outfit/emh //This gives him some NODROP items so he doesn't spawn naked like the EMHs of yore. The ID has preset access and cannot be removed. If they're being a little shit and greytiding, then ban them / execute them.
+	uniform = /obj/item/clothing/under/trek/medsci/ds9/emh
+	shoes = /obj/item/clothing/shoes/jackboots/emh
+	ears = /obj/item/radio/headset/headset_med/emh
+	id = /obj/item/card/id/emh
+
+/obj/item/card/id/emh
```

Comment:
`/obj/item/card/id/silver`
Redundant, make subtype of above and remove redundant definitions